katsdptelstate/test/test_rdb_handling.py

```diff
@@ -121,16 +125,9 @@ def test_load_from_file(self):
         # Write data to file
         rdb_writer = RDBWriter(client=write_ts.backend)
         rdb_writer.save(self.filename)
-
-        # Load it back into some backend
-        read_ts = self.make_telescope_state()
-        read_ts.load_from_file(self.filename)
-        self.assertEqual(read_ts.keys(), [b'immutable', b'mutable'])
-        self.assertTrue(read_ts.is_immutable('immutable'))
-        self.assertEqual(read_ts['immutable'], ['some value'])
-        self.assertEqual(
-            read_ts.get_range('mutable', st=0),
-            [('first', 12.0), ('second', 15.5)])
+        # Check loading from filenames and file-like objects
+        self.load_from_file_and_check(self.filename)
+        self.load_from_file_and_check(open(self.filename, 'rb'))
```

Comment:
Use a `with` statement to open the file to ensure the file handle doesn't hang around.

Reply:
Done.